ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/radiobutton/v1/radiobutton/clientlibs/site/js/radiobuttonview.js

```diff
@@ -76,14 +76,16 @@
             });
         }
 
-        _updateEnabled(enabled) {
+        _updateEnabled(enabled, readOnly) {
             this.toggle(enabled, FormView.Constants.ARIA_DISABLED, true);
             this.element.setAttribute(FormView.Constants.DATA_ATTRIBUTE_ENABLED, enabled);
             let widgets = this.widget;
             widgets.forEach(widget => {
                 if (enabled === false) {
                     widget.setAttribute(FormView.Constants.HTML_ATTRS.DISABLED, true);
                     widget.setAttribute(FormView.Constants.ARIA_DISABLED, true);
+                } if (readOnly == true) {
+                    widget.setAttribute(FormView.Constants.ARIA_READONLY, true);
                 } else {
```

Comment:
same issue as mentioned in https://github.com/adobe/aem-core-forms-components/pull/338/files#r1029062758